contracts/apps/IAragonApp.sol

```diff
@@ -0,0 +1,13 @@
+/*
+ * SPDX-License-Identifier:    MIT
+ */
+
+pragma solidity ^0.4.24;
+
+import "../kernel/IKernel.sol";
+
+
+contract IAragonApp {
+    function kernel() public view returns (IKernel);
+    function appId() public view returns (bytes32);
```

Comment:
Should we leverage this change and make these `external`?

Reply:
Why? Those functions are being called by other contracts and in Solidity 4 we can’t override from `external` to `public`.

Reply:
I don't see a problem in introducing a breaking change if we are going to use semver properly to release a breaking version

Reply:
Ok, but what’s the advantage of making them `external`?

Reply:
Well, it is the expected way to define interfaces in solc... you won't notice a warning here because it is defined as a `contract`, but otherwise it would complain. However, not a blocker at all, we still need to define what do we want to include in the next breaking version, so we can discuss whether to address this or not